pyam/core.py

```diff
@@ -311,21 +311,25 @@ def as_pandas(self, with_metadata=False):
         """
         if with_metadata:
             if isinstance(with_metadata, dict):
```

Comment:
now this can be a one-liner:

```
cols = self._discover_meta_cols(**with_metadata) if isinstance(with_metadata, dict) else self.meta.columns
```
```

Reply:
well, two lines or >80 ;-)